pages/components/block.tsx

```diff
@@ -0,0 +1,74 @@
+import { NextPage } from 'next';
+import Head from 'next/head';
+import { useTranslation } from 'next-export-i18n';
+import SideNavLayout from '../../layouts/SideNavLayout/SideNavLayout';
+import { navItems } from '../../utils/components-sidenav';
+import InfoBox from '../../components/InfoBox/InfoBox';
+import ComponentExample from '../../components/ComponentExample/ComponentExample';
+import {
+  Block,
+  Heading,
+  Text,
+  Paragraph,
+  ExternalLink,
+} from 'suomifi-ui-components';
+
+const Page: NextPage = () => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <Head>
+        <title>{t('block_page.site_title')}</title>
```

Comment:
Maybe the Block page could have some clearer examples regarding how to use the component as layout spacer. Using the top/bottom margin props for example